src/Generator/ServiceProviderGenerator.php

```diff
@@ -219,9 +220,19 @@ private function generateBodySerializerClosure(Specification $specification): Cl
             );
         }
 
+        if (in_array(VdnApiJsonContentTypeSerializer::MIME_TYPE, $allContentTypes, true)) {
+            $vdnApiJsonContentTypeSerializer = $this->builder->methodCall(
```

Comment:
I think we should keep the `Init` suffix for consistency.

Reply:
```suggestion
            $vdnApiJsonSerializerInit = $this->builder->methodCall(
```
perhaps

Reply:
fixed, thank you